client/client.go

```diff
@@ -5,6 +5,12 @@ import (
 	wcm "github.com/voidshard/wysteria/common/middleware"
 )
 
+const (
+	FACET_COLLECTION = "collection"
+	FACET_ITEM_TYPE = "itemtype"
+	FACET_ITEM_VARIANT = "variant"
```

Comment:
There are probably more you will end up needing to change, outside of the changes in this merge request. Yea?